### DISCUSSION TITLE
Does v11 Avalonia support net8.0 preview?

### DISCUSSION THREAD

**Discussion by cirrusone** [AvaloniaUI/Avalonia#11185] [Q&A] [answered]
I've searched the issues for 'net8' and cannot find any info. Does net8.0 preview work with Avalonia 11.0.0 Preview 7 ?

**Comment by MikeCodesDotNET** [comment, marked as answer]
![CleanShot 2023-04-29 at 13 26 22@2x](https://user-images.githubusercontent.com/552074/235300123-ca4b27dc-8911-4127-a905-81ba71e94d74.png)

Yep. No issues here :)
